src/main/scala/org/broadinstitute/dsde/agora/server/ga4gh/ModelSupport.scala

```diff
@@ -101,32 +99,4 @@ object ModelSupport {
     Metadata(version = version, `api-version` = apiVersion, country = country, `friendly-name` = friendlyName)
   }
 
-  /**
-   * Looks for all populated "meta: author=X" and "meta: email=Y" fields in the optional wdl meta fields.
-   */
-  def findAuthorsInWdl(wdl: Option[WdlNamespaceWithWorkflow]): String = {
-    val authorField = "author"
-    val emailField = "email"
-    val authors: List[String] = wdl match {
-      case Some(parsed) =>
-        parsed.tasks.map { task =>
-          formatAuthorEmail(task.meta.getOrElse(emailField,""), task.meta.getOrElse(authorField,""))
-        }.toList ++ parsed.workflows.map { workflow =>
-          formatAuthorEmail(workflow.meta.getOrElse(emailField,""), workflow.meta.getOrElse(authorField,""))
-        }
-      case _ => List.empty[String]
-    }
-    authors.filterNot(_.isEmpty).distinct.mkString(", ")
-  }
-
-  private def formatAuthorEmail(email: String, author: String): String = {
-    val stringFormat = "%s <%s>"
-    (email.trim.isEmpty, author.trim.isEmpty) match {
-      case (true, false) => author.trim
-      case (false, true) => email.trim
-      case (false, false) => stringFormat.format(author.trim, email.trim)
-      case _ => ""
-    }
-  }
-
```

Comment:
pull the author fixes into a separate PR and get that merged sooner?

Reply:
I'm hoping to get this merged soon :D